scripts/creation-portal-migration/api-list/program.js

```diff
@@ -0,0 +1,124 @@
+const { default: axios } = require("axios");
+const { request } = require("chai");
+const { CONFIG } = require("../constant/config");
+const { getHeaders } = require("./headers");
+
+
+
```

Comment:
@swetha-aggidevara As per our discussion if you can add comments related to functions that will be helpful